src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" >
```

Comment:
is there a specific reason you chose bootstrap? if its only for the grid system you could very well use css grid

Reply:
NO . There is no specific reason for using bootstrap. I will try with css grid.